nncf/quantization/algorithms/channel_alignment/algorithm.py

```diff
@@ -433,18 +428,23 @@ class ConvParamsContainer:
     Convolution container class which is incapsulating common convolutional parameters collection.
     """
 
-    def __init__(self, conv_op, model, nncf_graph, backend_entity: ChannelAlignmentAlgoBackend):
+    def __init__(
+        self, conv_op: NNCFNode, model: TModel, nncf_graph: NNCFGraph, backend_entity: ChannelAlignmentAlgoBackend
+    ):
         """
-        :param conv_op: Backend-specific conv node.
+        :param conv_op: NNCF conv node.
         :param model: Backend-specific model instance.
         :param nncf_graph: NNCFGraph of given backend-specific model.
         :param backend_entity: Current backend entity to retrieve parameters from given conv node
         """
         _, self._weights_port_id = backend_entity.get_weights_port_ids_for_node(conv_op)
         self.stated_weight = StatedTensor(backend_entity.get_weight_value(conv_op, model, self._weights_port_id))
-        bias = None
+        self._bias_op_exist = False
         if backend_entity.is_node_with_bias(conv_op, nncf_graph):
             bias = backend_entity.get_bias_value(conv_op, model, nncf_graph)
+            self._bias_op_exist = True
+        else:
+            bias = backend_entity.create_bias_tensor(conv_op, nncf_graph, 0)
```

Comment:
Why do we need the `self._bias_op_exist` variable if we create bias op anyway?

Reply:
We are using this information to create valid command over here: https://github.com/openvinotoolkit/nncf/pull/2073/files#diff-f1adfa678788a5eff3b6926406a5f3dd9ea63b1c510018485a3b1dce8931f110R152-R155
either we need to update weights or we need to create a new add op

Reply:
I keep this logic inside the algo to make model transformer as simple as it possible